### PR TITLE
feat(webapp): add hint text to create and edit a post of a group aside of the save and abort buttons

### LINKED ISSUE #6068: 🚀 [Feature] Webapp – Groups – On Create or Edit Page have Additional Hint `This post will be only visible in …`
## 🚀 Feature

Webapp – Groups – On Create or Edit Page have Additional Hint `This post will be only visible in Group XXX`.

### Todo

Add the text asside the `Save` button.

Bildschirmfoto 2023-03-02 um 13 04 47

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Add hint text to create and edit a post of a group aside of the save and abort buttons.

### Issues

- fixes #6068

### Todo

- [x] may optimize the flex behaviour of the buttons to be right aligned, especially visible in EN

### CODE REVIEW

File: webapp/components/ContributionForm/ContributionForm.vue [hunk 1]
```diff
@@ -64,14 +64,33 @@
...
+              danger
+            >
+              {{ $t('actions.cancel') }}
+            </base-button>
+          </ds-flex-item>
+          <ds-flex-item style="align-self: flex-end">
+            <base-button type="submit" icon="check" :loading="loading" :disabled="errors" filled>
+              {{ $t('actions.save') }}
+            </base-button>
+          </ds-flex-item>
```
Comment (Elweyn) [inline webapp/components/ContributionForm/ContributionForm.vue:92]: Is not optimal yet
Bildschirmfoto 2023-03-13 um 14 40 47

File: webapp/components/ContributionForm/ContributionForm.vue [hunk 2]
```diff
@@ -64,14 +64,33 @@
...
+            <base-button
+              data-test="cancel-button"
+              :disabled="loading"
+              @click="$router.back()"
+              danger
+            >
+              {{ $t('actions.cancel') }}
+            </base-button>
+          </ds-flex-item>
+          <ds-flex-item style="align-self: flex-end">
```
Comment (Elweyn) [inline webapp/components/ContributionForm/ContributionForm.vue:88]: Maybe just remove this

**Review by Elweyn — DISMISSED** [review]

File: webapp/components/ContributionForm/ContributionForm.vue (hunk 2 above)
Comment (Elweyn) [reply webapp/components/ContributionForm/ContributionForm.vue:88]:
```suggestion
```

File: webapp/components/ContributionForm/ContributionForm.vue [hunk 3]
```diff
@@ -64,14 +64,33 @@
...
-          </base-button>
-          <base-button type="submit" icon="check" :loading="loading" :disabled="errors" filled>
-            {{ $t('actions.save') }}
-          </base-button>
-        </div>
+        <ds-flex class="buttons-footer" gutter="xxx-small">
+          <ds-flex-item width="3.5">
+            <!-- eslint-disable vue/no-v-text-v-html-on-component -->
+            <ds-text
+              v-if="groupId"
```
Comment (Mogge) [inline webapp/components/ContributionForm/ContributionForm.vue:71]:
```suggestion
              v-if="showGroupHint"
```

File: webapp/components/ContributionForm/ContributionForm.vue [hunk 4]
```diff
@@ -152,6 +171,12 @@ export default {
     contentLength() {
       return this.$filters.removeHtml(this.formData.content).length
     },
+    groupId() {
+      return this.group && this.group.id
+    },
```
Comment (Mogge) [inline webapp/components/ContributionForm/ContributionForm.vue:176]:
```suggestion
    groupId() {
      return this.group && this.group.id
    },
    showGroupHint() {
      return this.grouupId && ['closed', 'hidden'].includes(this.group.groupType)
    },
```

**Review by Mogge — CHANGES_REQUESTED** [review]
We must also check for the group type. Posts in a public group are visible to everyone

File: webapp/components/ContributionForm/ContributionForm.vue (hunk 4 above)
Comment (Tirokk) [reply webapp/components/ContributionForm/ContributionForm.vue:174]: Big thanks for your important hint! @Mogge

File: webapp/components/ContributionForm/ContributionForm.vue (hunk 3 above)
Comment (Tirokk) [reply webapp/components/ContributionForm/ContributionForm.vue:71]: same same

File: webapp/components/ContributionForm/ContributionForm.vue (hunk 2 above)
Comment (Tirokk) [reply webapp/components/ContributionForm/ContributionForm.vue]: Thanks @Elweyn that you have done so and refined it!

File: webapp/components/ContributionForm/ContributionForm.vue (hunk 1 above)
Comment (Tirokk) [reply webapp/components/ContributionForm/ContributionForm.vue]: Done

**Pull request merged**